Sources/Stevenson/Services/SlackService.swift

```diff
@@ -96,14 +96,29 @@ public struct SlackCommandMetadata: Content {
             channelName:    container.decode(String.self, forKey: .channelName),
             command:        container.decode(String.self, forKey: .command),
             text:           container.decode(String.self, forKey: .text),
-            responseURL:    container.decode(String.self, forKey: .responseURL)
+            responseURL:    container.decodeIfPresent(String.self, forKey: .responseURL)
         )
     }
 }
 
 
 // MARK: Service
 
+extension Application {
+    public var slack: SlackService? {
+        get {
+            self.storage[SlackServiceKey.self]
+        }
+        set {
+            self.storage[SlackServiceKey.self] = newValue
+        }
+    }
+}
+
+struct SlackServiceKey: StorageKey {
```

Comment:
This `StorageKey` is a new way of storing dependencies/configuratino for Vapor now, right?

Reply:
Yep, if a service has a state (aka has a configuration with tokens, keys), this is how it should be set up based on docs